editor/editor_help.cpp

```diff
@@ -1379,7 +1379,16 @@ void EditorHelp::_update_doc() {
 		}
 
 		// Enums
+		bool has_enums = false;
 		if (enums.size()) {
+			for (KeyValue<String, DocData::EnumDoc> &E : cd.enums) {
+				if (!E.key.begins_with("_") || !E.value.description.strip_edges().is_empty()) {
+					has_enums = true;
+					break;
+				}
```

Comment:
It makes sense to check `is_script_doc`, like here:

https://github.com/godotengine/godot/blob/447030ec5efc44f8edf66f84065f9d45a344d3a8/editor/editor_help.cpp#L983-L986

Reply:
can you check the latest commit to see if it is correct?